spec/spec_helper.rb

```diff
@@ -40,12 +43,12 @@ module SpecHelper
   # We should take their advice!
   config.raise_on_warning = true
 
-  config.include(ExpectOffense)
+  config.include RuboCop::RSpec::ExpectOffense
 
   config.include_context 'with default RSpec/Language config', :config
   config.include_context 'smoke test', type: :cop_spec
```

Comment:
I don’t think we need these two lines.

Reply:
RSpec Rails is also RSpec, so I thought it would be nice to have RSpec/Language loaded and smoke_test for RSpec. What do you think?